content/en/docs/instrumentation/java/extensibility.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Extensibility
+description: Extensions add new features and capabilities to the agent without having to create a separate distribution or to fork this repository. For example, you can create custom samplers or span exporters, set new defaults, and embed it all in the agent to obtain a single jar file.
+---
+
+## Introduction
+
+Extensions add new features and capabilities to the agent without having to create a separate distribution (for examples and ideas, see [Use cases for extensions](#sample-use-cases)).
+
+The contents in this folder demonstrate how to create an extension for the OpenTelemetry Java instrumentation agent, with examples for every extension point. 
+
+> Read both the source code and the Gradle build script, as they contain documentation that explains the purpose of all the major components.
+
+## Build and add extensions
+
+To build this extension project, run `./gradlew build`. You can find the resulting jar file in `build/libs/`. 
```

Comment:
This line says "run `./gradlew build`" without any context. What steps do I need to following _before_ I can run this command? I assume that there are sources to fetch first.

Reply:
I removed that section, will remain in the repo.